Sources/ShortcutRecorder/SRRecorderControlStyle.m

```diff
@@ -320,8 +322,8 @@ - (NSComparisonResult)compare:(SRRecorderControlStyleComponents *)anOtherCompone
          relativeToComponents:(SRRecorderControlStyleComponents *)anIdealComponents
 {
     static NSDictionary<NSNumber *, NSArray<NSNumber *> *> *AppearanceOrderMap = nil;
-//    static NSDictionary<NSNumber *, NSArray<NSNumber *> *> *TintOrderMap = nil;
-//    static NSDictionary<NSNumber *, NSArray<NSNumber *> *> *DirectionOrderMap = nil;
+    static NSDictionary<NSNumber *, NSArray<NSNumber *> *> *TintOrderMap = nil;
+    static NSDictionary<NSNumber *, NSArray<NSNumber *> *> *DirectionOrderMap = nil;
```

Comment:
These were commented out by jnozzi in af4f0b0617 but there’s no detail on exactly what warnings it was giving. I restored this and I see no warnings, so let’s keep with upstream.